app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -69,21 +74,33 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
         Parser.parseVariables(application)
 
         LeanplumActivityHelper.enableLifecycleCallbacks(application)
-
-        val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
-
-        Leanplum.start(application, hashMapOf(
-            "default_browser" to MozillaProductDetector.getMozillaBrowserDefault(application).orEmpty(),
-            "fennec_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.FIREFOX.productName),
-            "focus_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.FOCUS.productName),
-            "klar_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.KLAR.productName),
-            "fxa_signed_in" to application.settings().fxaSignedIn,
-            "fxa_has_synced_items" to application.settings().fxaHasSyncedItems,
-            "search_widget_installed" to application.settings().searchWidgetInstalled
-        ))
+        Leanplum.start(application)
+
+        lpmsSetUserAttributeJob = MainScope().launch {
```

Comment:
I'm not sure if this runs off the main thread since you're using MainScope. I think here it would be better to have a scope property on the service and then launch with the IO Dispatcher.

```
class LeanplumMetricsService {
  private val scope = CoroutineScope()

  start() {
    scope.launch(IO) { ... }
  }

  stop() {
    runBlocking { job.join() }
    scope.cancel()
  }